app/javascript/dashboard/components/widgets/conversation/MoreActions.vue

```diff
@@ -97,7 +81,14 @@ export default {
 };
 </script>
 <style scoped lang="scss">
-@import '~dashboard/assets/scss/mixins';
+.actions--container {
+  align-items: center;
+  .button {
```

Comment:
Is it possible to stick with component provided styles via props ?

Reply:
@nithindavid The color for secondary is too dark and attention-grabbing so it is hard to use, please suggest and alternative.

Reply:
Got it, I have added TODO on my side. Will take another iteration for button styles.